spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesServiceInstance.java

```diff
@@ -37,43 +34,49 @@
 
 	private static final String DSL = "//";
 
-	private static final String COLN = ":";
+	private static final String COLON = ":";
+
+	private String instanceId;
 
-	private final String instanceId;
+	private String serviceId;
```

Comment:
`serviceId` and `instanceId` are terms used throughout the Spring Cloud stack, so let's keep them here the way they were.

Reply:
Done